readme.md

```diff
@@ -0,0 +1,23 @@
+How to build
+------------
+
+Things you'll need to develop and test abracadabra:
+
+* [Node.js and npm](https://nodejs.org/en/download/)
+* [Yarn](https://yarnpkg.com/en/docs/install)
```

Comment:
Mostly as a comment - I think the mention in itself is fine: I think that currently it's up to personal taste whether using one uses npm or yarn. Projects should work equally well nowadays, as long as you are consistent and only use one package manager throughout. This is actually the reason I opted to use `npx` in the scripts, it works equally well both ways.

That being said, I use Yarn, so that's what I know will work with this.